src/SignalR/common/SignalR.Common/ref/Microsoft.AspNetCore.SignalR.Common.csproj

```diff
@@ -5,12 +5,11 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <Compile Include="Microsoft.AspNetCore.SignalR.Common.netstandard2.0.cs" />
-    <Reference Include="Microsoft.AspNetCore.Connections.Abstractions" />
-    <Reference Include="Microsoft.Extensions.Options" />
-    <Reference Include="System.Buffers" />
-    <Reference Include="System.Text.Json" />
+    <Reference Include="Microsoft.AspNetCore.Connections.Abstractions"  />
```

Comment:
Whats with all the extra spaces in the references?

Reply:
That's just the way the generator works and it's not our generator. In any case GenApi isn't about pretty files 😺

Reply:
Oh it's the ref file, carry on 😄